### ISSUE TITLE
cant install via pnpm

### ISSUE THREAD

**Issue by toybreaker** [TheOtterlord/astro-csv#1]
hey, thanks for making this, it would be useful, if just i could install it!
i'm using PNPM.

This is the error i get:

```
|>pnpx astro add astro-csv
../../.pnpm-store/v3/tmp/dlx-10721       | +409 +++++++++++++++++++++++++++++++++++++++++
../../.pnpm-store/v3/tmp/dlx-10721       | Progress: resolved 430, reused 409, downloaded 0, added 409, done
⚠ astro-csv is not an official Astro package. Use at your own risk!
✔ Continue? … yes
✖ astro-csv doesn't appear to be an integration or an adapter. Find our official integrations at https://astro.build/integrations
 ERROR  Command failed with exit code 1: astro add astro-csv

pnpm: Command failed with exit code 1: astro add astro-csv
    at makeError (/opt/homebrew/Cellar/pnpm/8.6.0/libexec/dist/pnpm.cjs:24242:17)
    at handlePromise (/opt/homebrew/Cellar/pnpm/8.6.0/libexec/dist/pnpm.cjs:24813:33)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.handler [as dlx] (/opt/homebrew/Cellar/pnpm/8.6.0/libexec/dist/pnpm.cjs:223506:7)
    at async /opt/homebrew/Cellar/pnpm/8.6.0/libexec/dist/pnpm.cjs:232908:21
    at async main (/opt/homebrew/Cellar/pnpm/8.6.0/libexec/dist/pnpm.cjs:232875:34)
    at async runPnpm (/opt/homebrew/Cellar/pnpm/8.6.0/libexec/dist/pnpm.cjs:233130:5)
    at async /opt/homebrew/Cellar/pnpm/8.6.0/libexec/dist/pnpm.cjs:233122:7

```

Am i doing something wrong?

**Comment by ptc-tonyo** [comment]
Confirmed as having the same issue. Can't use this handy integration.

**Comment by theotterlord** [comment]
Fixed in v0.2.1. I've also included manual installation instructions in the README, but it should now work through `astro add`. Let me know if problems persist!
